### PR TITLE
Api documentations for Activity and Location

### LINKED ISSUE #1: Locations
- [x] #5
- [ ] Document tests
- [ ] Implement tests
- [ ] Implement API

### LINKED ISSUE #5: Locations document API

### PR DESCRIPTION
**Pull request by cockleshell**

### CODE REVIEW

File: app/README.md [hunk 1]
```diff
@@ -1,23 +1,63 @@
...
-        },
-        {
-            "title": "Support",
-            "id": "02",
-            "imageUrl":"http://google.com/tst.jpg"
-        }
-    ]
-}
-```
+## **How to start server and tests**
```
Comment (mepoohsta) [inline app/README.md:3]: ` and tests`, maybe remove?

File: app/README.md [hunk 2]
```diff
@@ -1,23 +1,63 @@
...
+
+* **Method:**
+
+  `GET`
+  
+*  **URL Params**
+
+   **Required:**
+  
+    None
```
Comment (mepoohsta) [inline app/README.md:28]: Do you need language as required or optional param?

File: app/README.md [hunk 3]
```diff
@@ -1,23 +1,63 @@
...
+ 
+   `activity_id = [integer]`
+
+* **Data Params**
+
+  None
+
+* **Success Response:**
+
+  * **Code:** 200
```
Comment (mepoohsta) [inline app/README.md]: Maybe 404 if activity was not found?
Maybe 422 if required parameter was not specified?

File: app/README.md [hunk 4]
```diff
@@ -1,23 +1,63 @@
...
+
+* **Method:**
+
+  `GET`
+  
+*  **URL Params**
+
+   **Required:**
+ 
+   `activity_id = [integer]`
```
Comment (mepoohsta) [inline app/README.md]: Maybe a little description for the param?

File: app/README.md [hunk 5]
```diff
@@ -1,23 +1,63 @@
...
+  * **Code:** 200
+
+
+### **Get api/locations**
+
+  Returns json data about available locations.
+
+* **URL**
+
+  /api/location?activity_id = [integer]
```
Comment (mepoohsta) [inline app/README.md]: Maybe you don't need to display param here? 

You described them in URL Params section.

File: app/README.md (hunk 2 above)
Comment (mepoohsta) [inline app/README.md]: Do you need language as required or optional param?

File: app/README.md (hunk 1 above)
Comment (mepoohsta) [inline app/README.md]: `and tests`, maybe remove?

**Review by mepoohsta — CHANGES_REQUESTED** [review]

File: app/README.md (hunk 5 above)
Comment (mepoohsta) [inline app/README.md]: Also `location` => `locations`

File: app/README.md (hunk 1 above)
Comment (cockleshell) [reply app/README.md:3]: removed

File: app/README.md (hunk 2 above)
Comment (cockleshell) [reply app/README.md]: I will ask Yurii later

File: app/README.md (hunk 5 above)
Comment (cockleshell) [reply app/README.md]: change

File: app/README.md (hunk 4 above)
Comment (cockleshell) [reply app/README.md:55]: good idea

File: app/README.md (hunk 1 above)
Comment (cockleshell) [reply app/README.md]: removed

File: app/README.md (hunk 2 above)
Comment (cockleshell) [reply app/README.md]: I add language as required

File: app/README.md (hunk 5 above)
Comment (cockleshell) [reply app/README.md]: good idea

File: app/README.md (hunk 4 above)
Comment (cockleshell) [reply app/README.md]: add

File: app/README.md (hunk 3 above)
Comment (cockleshell) [reply app/README.md]: add

File: app/README.md [hunk 6]
```diff
@@ -1,23 +1,156 @@
...
-{
-    "data": [
+### Run Server
+
+Go to */app* and use command **yarn run start**
+
+
+## **API routes**
+
+### **GET /api/activities**
```
Comment (mepoohsta) [inline app/README.md]: My suggestion to name it `List of activities`

File: app/README.md [hunk 7]
```diff
@@ -1,23 +1,156 @@
...
+        {
+          "code" : "LANGUAGE_NOT_FOUND",
+          "message" : "Language is not provided"
+        }
+      ]
+    }
+    ```
+
+
+### **Get api/locations**
```
Comment (mepoohsta) [inline app/README.md]: My suggestion to name it `List of locations`

File: app/README.md [hunk 8]
```diff
@@ -1,23 +1,156 @@
...
+   **Required:**
+
+   `activity_id = [integer]`
+
+   `language = [string]`
+
+* **Success Response:**
+
+  * **Code:** 200
+    **Content:**
```
Comment (mepoohsta) [inline app/README.md:100]: You need to add blank space here

Comment (cockleshell) [reply app/README.md:100]: Added blank space

**Review by mepoohsta — APPROVED** [review]

**Pull request merged**